handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -1102,6 +1103,7 @@ protected static KeyStore buildKeyStore(X509Certificate[] certChain, PrivateKey
                                   char[] keyPasswordChars, String keyStoreType)
             throws KeyStoreException, NoSuchAlgorithmException,
                    CertificateException, IOException {
+        ObjectUtil.checkNotNull(certChain, "certChain");
```

Comment:
The `setKeyEntry` will throw an `IllegalArgumentException` saying "private key must be accompanied by a cert. chain" which I like better than this generic looking NullPointerException.

Reply:
thats a good point here... @skyguard1 let's just revert this one and keep the rest.

Reply:
I've done it